### PR TITLE
Approved list & Pathfinder 2 v2.01

### PR DESCRIPTION
**Pull request by clevett**
## Changes / Comments

Release notes - User Collaboration - Version Pathfinder 2 v2.01

** Task
    * [UC-675] - Pathfinder 2: Refactor Sheetworkers
    * [UC-698] - Pathfinder 2: Game Settings set default sheet type

** Bug
    * [UC-699] - Pathfinder 2:  Move NPC setting buttons to the left side
    * [UC-701] - Pathfinder 2: Saving throws roll template calls wrong notes attribute
    * [UC-710] - Pathfinder 2: Display fixes
    * [UC-712] - Pathfinder 2: Wrong spell DC calculation when magic tradition is selected

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

File: approved.yaml [hunk 1]
```diff
@@ -70,16 +70,24 @@ thestrange:
...
-  - "Earthdawn"
-
 bandofbladesofficial:
   - "Band of Blades Official"
   - "Band of Blades by Evil Hat Productions"
-  - "Blades in the Dark"
+  - "Forged in the Dark"
 
+bladesinthedarkadvancedofficial:
+- "Blades in the Dark"
```
Comment (joesinghaus) [inline approved.yaml]: @clevett what's the reason for this change? FYI, this has broken every single game for Blades (and S&V too, I presume) by forcing users to reselect sheet in game settings (and it's not obvious they need to do that).

Comment (clevett) [reply approved.yaml]: @joesinghaus 

*official* is required in the short name to set the **sheet by Publisher**.  An oversight on my part as you're right this results in the sheet no longer being selected. I'll need to make an adjustment in the future to better handle these updates.

Comment (joesinghaus) [reply approved.yaml]: @clevett Right, no worries, there were just a lot of not-so-tech-savvy people confused as to where their character sheet went. Maybe in the future it would be a good idea to migrate the selected sheet for all existing users along with such a change.
